backend/src/test/groovy/com/capstone/moneytree/utils/MoneyTreeTestUtils.groovy

```diff
@@ -20,23 +25,6 @@ class MoneyTreeTestUtils {
               .lastName(lastName)
               .alpacaApiKey(alpacaApiKey)
               .build()
-      user.setId(new Random().nextLong())
-      return user
-   }
-
-   /**
-    * Utility method to create a user
-    */
-   static User createUser(Long id, String email, String username, String password, String firstName, String lastName, String alpacaApiKey) {
```

Comment:
This was removed so that we can use createUser(email, ....) and setId afterwards if I understand, correct?

Reply:
Yes, I will explain why.

The IDs are autogenerated by the spring OGM. if we use that utility method and provide the Ids, the database persistence fails. We really need to make sure that we use the generated Ids from ogm.

I would rather have them generated by ogm then setting it up afterwards to not break persistence.

I discovered that when trying to refactor the tests by providing my own ids.

Reply:
Perfect! Sounds good to me.